great_expectations/__init__.py

```diff
@@ -5,6 +5,8 @@
 
 from great_expectations.data_context.migrator.cloud_migrator import CloudMigrator
 
+from .expectations.registry import register_core_expectations
```

Comment:
Why is this a relative import?

Reply:
@Kilo59 Because that is how `PyCharm` made it when I used the tool's feature.  Should I make it absolute level?  Thanks.

Reply:
@Kilo59 I made the change.  Thanks.